src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -146,6 +146,15 @@ std::string Notification::toJson(const std::string& attrsFormat)
 
   jh.addString("attrsFormat", attrsFormat);
 
+  if (this->httpInfo.payload.empty() && this->includePayload)
+  {
+    jh.addBool("includePayload", true);
+  }
+  else
+  {
+    jh.addBool("includePayload", false);
+  }
+
```

Comment:
This fragment should be moved to HttpInfo::toJson(), excluding `this->httpInfo.payload.empty()` from the if condition. Note that the new field shouldd be inside `httpCustom` but not in `notification` directly.

Reply:
Updated